src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString
+    .trim()
+    .split(';')
+    .filter((str) => str.trim())
+    .map((str) => {
+      return str
+        .trim()
+        .split(':')
+        .map((s) => s.trim());
```

Comment:
The use of `filter` and `map` is correct as they do not mutate the original array. However, consider combining the two `map` calls into one for efficiency and readability.